README.md

```diff
@@ -1,2 +1,90 @@
-# kruize-vpa
-Kruize VPA Integration
+# Kruize - VPA
+
+Kruize VPA Recommender for Kubernetes/ Openshift
```

Comment:
Give a reference to clever saying that this code is based off of the clever example

Reply:
Updated the README.md file